### DISCUSSION TITLE
尝试通过 zbp.exe -c config.json 命令来读取配置文件时报错

### DISCUSSION THREAD

**Discussion by echothesky** [FloatTech/ZeroBot-Plugin#558] [Q&A] [answered]
![屏幕截图 2023-01-10 113531](https://user-images.githubusercontent.com/106248959/211456498-8e29541f-8114-4707-9c13-b97fb241775f.png)

**Comment by kanrichan** [comment, marked as answer]
检查配置文件是否符合 `Json` 格式

**Reply by echothesky** [reply]
json文件没有任何问题，并且以前也是这般配置也不会出现问题

**Reply by kanrichan** [reply]
请贴上你的配置文件截图

**Reply by kanrichan** [reply]
或者请用其他语言对配置文件执行解析并附上相关截图

**Reply by echothesky** [reply]
![屏幕截图 2023-01-11 111220](https://user-images.githubusercontent.com/106248959/211709554-20f03dd1-2c2c-43f3-bd77-1ff26bfa8ffa.png)
![屏幕截图 2023-01-11 111203](https://user-images.githubusercontent.com/106248959/211709563-9e6026a8-a057-4f5b-be10-b6901383b5c8.png)
配置文件如上图

**Reply by echothesky** [reply]
![屏幕截图 2023-01-11 111705](https://user-images.githubusercontent.com/106248959/211709901-2d0f9731-24e5-4f0e-aaed-4eb4fe520fc8.png)
![屏幕截图 2023-01-11 111644](https://user-images.githubusercontent.com/106248959/211709916-23cdd670-0080-4d1c-a47e-59a6dd2c9969.png)
在进行unicode转码之后问题依旧没有解决，unicode转码过后的配置文件如上

**Reply by echothesky** [reply]
该问题仅出现于windows server 2016的云服务器之上，在自己本地电脑设备上进行了多个账号同时登录gocq与zerobot均未出现问题，问题出现原因会不会与云服务器环境有关
